src/documents/actor/character.ts

```diff
@@ -410,6 +412,32 @@ export class NimbleCharacter extends NimbleBaseActor {
 		await this.#dialogs.configureSkills.render(true);
 	}
 
+	async configureHitPoints() {
+		const dialog = new GenericDialog(
+			`${this.name}: Configure Hit Points`,
+			EditHitPointsDialog,
+			{ document: this },
+			{ icon: 'fa-solid fa-heart', width: 250 },
+		);
+
+		await dialog.render(true);
+		const result = await dialog.promise;
+
```

Comment:
Can just return early if no result. That way don't have to nest everything in an if block